src/api/routers/participantsRouter.spec.ts

```diff
@@ -0,0 +1,196 @@
+import { Response } from 'express';
+import { http, HttpResponse } from 'msw';
+import { setupServer } from 'msw/node';
+
+import { TestConfigure } from '../../database/TestSelfServeDatabase';
+import { Participant, ParticipantStatus } from '../entities/Participant';
+import { User } from '../entities/User';
+import { SSP_ADMIN_SERVICE_BASE_URL } from '../envars';
+import { KeyPairDTO } from '../services/adminServiceHelpers';
+import { ParticipantRequest } from '../services/participantsService';
+import { createParticipant, createResponseObject, createUser } from './helperFunctions';
+import { getParticipantKeyPairs, getParticipantUsers } from './participantsRouter';
+
+const oneKeyPair: KeyPairDTO[] = [
```

Comment:
In the process of splitting the router and test into separate files, perhaps the mock data could also be extracted into a separate file. We could possibly also use faker.js here